modules/lz/sub-vending/src/self/subResourceWrapper/deploy.bicep

```diff
@@ -80,6 +80,9 @@ param roleAssignmentEnabled bool = false
 @sys.description('Supply an array of objects containing the details of the role assignments to create.')
 param roleAssignments array = []
 
+@sys.description('Disable telemetry collection by this module. For more information on the telemetry collected by this module, that is controlled by this parameter, see this page in the wiki: [Telemetry Tracking Using Customer Usage Attribution (PID)](https://github.com/Azure/bicep-lz-vending/wiki/Telemetry)')
+param disableTelemetry bool = false
```

Comment:
Param only seem to be referenced by a variable that inverts the value.

Readability would be improved if the param was `enableTelemetry`.

Reply:
Hey @Gordonby,

This is so we dont have to introduce a breaking change as in the calling module `main.bicep` we have it the other way today.

Not something we want to change for breaking reasons, and no need to as we can handle as we have written in this PR

Reply:
Rationale seems clear.

Wonder if there's a wider telemetry guidance required for the bicep registry contributors. It would be great if we could define consistent param names for opting in/out of telemetry gathering that modules may do. I'll raise an issue to cover this concern and resolve this discussion.